src/PrestaShopBundle/Resources/views/Admin/Configure/ShopParameters/product_preferences.html.twig

```diff
@@ -0,0 +1,248 @@
+{#**
+ * 2007-2018 PrestaShop
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2018 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+{% extends '@PrestaShop/Admin/layout.html.twig' %}
+{% trans_default_domain "Admin.Shopparameters.Feature" %}
+
+{% set generalForm, paginationForm, pageForm, stockForm = form.general, form.pagination, form.page, form.stock %}
+
+{% block content %}
+    <div class="container">
+        {{ form_start(form, {'attr': {'class': 'form'}}) }}
+        <div class="row">
+            {% block product_preferences_general %}
```

Comment:
maybe can you create some template into `views/Admin/Configure/ShopParameters/Blocks` and use include? This make templates easier to be overridden and customized using bundles/modules.

It's not a strict rule, but since we've refactored Product Pages templates we try to keep template file under 100 lines :)

Reply:
Good idea. :) Updated template.